pages/osx/xattr.md

```diff
@@ -14,6 +14,10 @@
 
 `xattr -d {{attribute_key}} {{file}}`
 
+- Delete the quarantine attribute from a given file:
+
+`xattr -d com.apple.quarantine {{file}}`
```

Comment:
If deleting quarantine is the most common use, we can use it as a concrete example in the previous description itself. Thereby, we cite a practical and the most common use case for the delete attribute command and prevent adding another example.